lib/main.js

```diff
@@ -23,95 +112,40 @@ if (flags.get('version')) {
   var diag = require('./diagnostics');
   diag();
 
+} else if (flags.get('devices').length > 0) {
+	
+	var deviceArguments = flags.get('devices');
+	for(var i = 0; i < deviceArguments.length ; i++) {
+		
+		var deviceArg = deviceArguments[i];
+		
+		// set up a domain to catch errors from setting up individual
```

Comment:
Perhaps we should setup domain stuff inside of `setupSonos` if we're going to add them? That way both have indep. error handling.